src/Illuminate/Foundation/Vite.php

```diff
@@ -282,11 +282,23 @@ public function usePreloadTagAttributes($attributes)
     }
 
     /**
-     * Use the "waterfall" prefetching strategy.
+     * Eagerly prefetch assets.
      *
      * @param  int|null  $concurrency
      * @return $this
      */
+    public function prefetch($concurrency = null)
+    {
+        return $concurrency === null
+            ? $this->usePrefetchStrategy('aggressive')
+            : $this->usePrefetchStrategy('waterfall', ['concurrency' => $concurrency]);
```

Comment:
```suggestion
            ? $this->useAggressivePrefetching()
            : $this->useWaterfallPrefetching($concurrency);
```
While this works, it would've been a good idea to use these methods the class already provides.